BUILDING.md

```diff
@@ -15,3 +18,70 @@ Sometimes it might complain that the `libpq` library is not found. This might ha
 * `libpq` library is not installed at all. 
 * On some platforms setting the `ARCHFLAGS` env to `-arch x86_64` and then installing pg would work. 
   So run this command: `sudo ARCHFLAGS="-arch x86_64" gem install pg -v '<some version>'`
+
+# Maintenance
+This section contains package maintenance procedures in preparation for deployment.
+
+* Review, approve, and merge open pull requests into the master branch.
+* Tag the master branch with the new version. Use the flag `-s` to sign a tag, or `-a` to create an unsigned tag.
+  ```
+  git tag -s "0.4.0"
+  git push --tags
+  ```
+* Create a new release from the new tag on GitHub and let GitHub auto-generate release notes. Mark releases used for testing as pre-release. The release bundles the code into an archive, which we'll use for deployment.
+
+# Deployment
+This section contains instructions to deploy Build and the related service build2xlsform to a test or production server.
+
+## Deploy build2xlsform
+The symlink `/srv/xls_service/current` points to the unpacked and built production release.
+
+SSH into the relevant server. The server admin will have added your SSH pubkey into `authorized_keys`.
+
+```
+# Replace 1.6 with your release version
+export B2X="1.6"
+echo $B2X
+
+su - xls_service
+cd /srv/xls_service/releases
+wget https://github.com/getodk/build2xlsform/archive/$B2X.tar.gz
+tar -xf $B2X.tar.gz && mv build2xlsform-$B2X $B2X && rm $B2X.tar.gz
+cd $B2X
+make
+rm /srv/xls_service/current
+ln -s /srv/xls_service/releases/$B2X /srv/xls_service/current
+exit
```

Comment:
If you wanted to be consistent with the steps below, you could add...
```
# verify that current points to latest release:
ls -l /srv/xls_service/current
```

Reply:
Ah saw that too late on mobile, just merged. Will add in next change!